.github/workflows/cicd.yml

```diff
@@ -5,6 +5,16 @@ on:
     paths-ignore:
       - 'apps-rendering/**'
       - 'dotcom-rendering/docs/**'
+  pull_request:
+    branches:
+      - '*'
```

Comment:
Running on `push` and `pull_request` causes a doubling of builds 😱:

<img width="514" alt="image" src="https://github.com/guardian/dotcom-rendering/assets/836140/469087d9-1ff9-4cba-b877-d215fd433b39">